### PR TITLE
Syncany.SyncanyCLI version 0.4.9

### PR DESCRIPTION
**Pull request by superusercode**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109607&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
This package is a set of `BAT` files which call Java. Should a `JRE` be a dependency?
```

ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.
```

File: manifests/s/Syncany/SyncanyCLI/0.4.9/Syncany.SyncanyCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Syncany.SyncanyCLI
+PackageVersion: 0.4.9
+Installers:
```
Comment (stephengillie) [inline manifests/s/Syncany/SyncanyCLI/0.4.9/Syncany.SyncanyCLI.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Oracle.JavaRuntimeEnvironment
Installers:
```

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by stephengillie** [comment]
Hi @superusercode,

This package appears to consist solely of BAT and JAR files. I believe we're not currently accepting scripted applications. 
image

**Pull request closed without merging**